components/Alert/alert.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alerts</title>
+    <link rel="stylesheet" href="alert.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <!-- 
+        alert with backgraound
+     -->
+    <div class="alert-container">
+        <div class="alert-with-bckg">
+            <div class="alert-box alert-primary"><b>Primary!</b> This is A primary alert Space🚀 with a link!! <a href="#">CHECK IT OUT</a> </div>
+            <div class="alert-box alert-danger"><b>👿 Danger!</b> This is A Danger alert Space.    </div>
+            <div class="alert-box alert-success"><b>✅ Success!</b> This is A Success alert Space.    </div>
+            <div class="alert-box alert-warning"><b>⚠️ Warning!</b> This is A Waring alert Space.   </div>
+    </div>
+        <!-- 
+            alert with outline
+        -->
+       <div class="alert-outline">
+            <div class="alert-box alert-primary-outline"><b>Primary!</b> This is A primary alert Space🚀 with a link!! <a href="#">CHECK IT OUT</a> </div>
+            <div class="alert-box alert-danger-outline"><b>👿 Danger!</b> This is A Danger alert Space.    </div>
+            <div class="alert-box alert-success-outline"><b>✅ Success!</b> This is A Success alert Space.    </div>
+            <div class="alert-box alert-warning-outline"><b>⚠️ Warning!</b> This is A Waring alert Space.   </div>
+    </div>
+    <!-- 
+        alert with button
+     -->
+     <div class="alert-btn">
+            <div class="alert-box alert-warning"><b>⚠️ Warning!</b> This is A Waring alert Space.</div>
+
```

Comment:
you can erase space or you can use prettier to format code.

Reply:
ok try to use it from now on